src/lib.rs

```diff
@@ -27,9 +27,12 @@
 #![cfg_attr(all(target_os = "linux", target_arch = "arm"), doc(
     html_root_url = "https://doc.rust-lang.org/libc/arm-unknown-linux-gnueabihf"
 ))]
-#![cfg_attr(all(target_os = "linux", target_arch = "mips"), doc(
+#![cfg_attr(all(target_os = "linux", target_arch = "mips", target_env = "gnu"), doc(
     html_root_url = "https://doc.rust-lang.org/libc/mips-unknown-linux-gnu"
 ))]
+#![cfg_attr(all(target_os = "linux", target_arch = "mips", target_env = "musl"), doc(
```

Comment:
There's a match for `target_env = "musl"` below which may need the addition of `target_arch` as well